src/plugins/foxPage/components/Layout.stories.tsx

```diff
@@ -80,7 +82,12 @@ export const FoxLayout: Story = () => {
               direction={{ base: 'column', xl: 'row' }}
             >
               <Stack spacing={4} flex='1 1 0%' width='full'>
-                <Text>{'Fox Page'}</Text>
+                <MainOpportunity
+                  balance='--'
+                  tvl={bnOrZero(150000000)}
```

Comment:
This should be a string as well, let the component make it a BN, as they don't serialize well.
```suggestion
                  tvl={'150000000'}
```